pnpm-workspace.yaml

```diff
@@ -1,42 +1,86 @@
+# NOTE: The `devDeps` and `peerDeps` catalogs in this file are
+# generated from webapp's package.json. To update them, run:
+#
+#     utils/sync-dependencies.ts ../webapp/services/static/package.json
+#
```

Comment:
Could you expand this comment to explain how we manage dev and peer deps?  (basically much of what the PR description is).  That'll help folks managing this file to know how to update it.